.circleci/config.yml

```diff
@@ -0,0 +1,72 @@
+version: 2.1
+
+commands:
+  before_install:
+    steps:
+      - run: |
+          dnf install make -y
+
+  install:
+    steps:
+      - run:
+          name : list_folder
+          command : ls
+      - run:
+          name : install_cmd
+          command: make install
+  
+  before_script:
+    steps:
+      - run: |
+          sudo -u nuodb "/opt/nuodb/etc/nuoadmin" tls $NUO_SET_TLS
+          sudo -u nuodb "/opt/nuodb/etc/nuoadmin" start
+
+
+  script:
+    steps:
+      - run:
+          name: script_cmd
+          command: make test NUODB_HOME="/opt/nuodb"
+  
+  after_failure:
+    parameters:
+      when:
+        type: string
+    steps:
+      - run:
+          command: |
+            cat /var/log/nuodb/nuoadmin.log*
+            grep ssl /opt/nuodb/etc/nuoadmin.conf"
```

Comment:
There seems to be a missing quote on this line?

You really only need quotes when an argument either does, or could (by using a variable) contain special characters that you don't want to be interpreted.  So it's not really need to quote arguments like "/opt/nuodb" etc. which have only plain characters in them.

But it doesn't hurt, either.